doc/install.text

```diff
@@ -0,0 +1,124 @@
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
+* Install git-lfs (https://git-lfs.github.com/)
+* Install matplotlib:
+
+conda install matplotlib
+
+
+If you have a very fresh compiler version, this could lead to incompatibilities 
+with the precompiled LSST stack. In case you run into trouble compiling or testing
+the LSST stack please install your compiler in anaconda:
+
+conda install gcc
+
+
+* Create a directory for the PFS-DRP repositories:
+
+mkdir ~/stella-git
+cd ~/stella-git
+
+* Clone the relevant git repositories:
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+
+
+* Install a newer version of eigen:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Declare the git repositories in Linux (replace <Linux64> with <DarwinX86> for Mac OS X):
+
+eups declare obs_pfs 1.0 -c -f Linux64 -r ~/stella-git/obs_pfs
+eups declare drp_stella_data 1.0 -c -f Linux64 -r ~/stella-git/drp_stella_data
+eups declare drp_stella 1.0 -c -f Linux64 -r ~/stella-git/drp_stella
+
+
+* Setup the pipeline. As obs_pfs and drp_stella_data are required packages they are set up automatically:
+
+setup drp_stella 1.0 -v
+
+During this step the environment variables $DRP_STELLA_DIR, $DRP_STELLA_DATA_DIR, and $OBS_PFS_DIR are automatically 
+created as well:
+
+echo $DRP_STELLA_DIR
+
+should show you:
+
+~/stella-git/drp_stella
+
+
+* Build the pipeline:
+
+cd drp_stella
+scons opt=3 -j8
+
+
+* Create the defects database (done during compilation of the code):
+
+cd ../obs_pfs
+scons opt=3 -j8
+
+
+* Now for using the pipeline. Raw test data are in $DRP_STELLA_DATA_DIR/tests/data/raw/ (3 Biases, 3 Darks, 1 Flat, 1 Arc). 
+First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+Let's assume you have a directory ~/spectra and do everything related to spectra there:
+
+mkdir ~/spectra/PFS
+mkdir ~/spectra/PFS/CALIB
+
+
+* Configuration parameters can be passed to the pipeline task either on the command line or in a config file 
+(e.g. $OBS_PFS_DIR/config/pfs/ingest.py, see following example). You can list all possible configuration parameters by 
+appending a "--show config" to the parameter list.
+To ingest the raw images:
+
+cd ~/spectra
+ingestImages.py PFS $DRP_STELLA_DATA_DIR/tests/data/raw/*.fits --output=PFS --mode=link -C $OBS_PFS_DIR/config/pfs/ingest.py
+
+
+* You can now inspect the created SQL database with e.g. sqlitebrowser (http://sqlitebrowser.org/).
+Now that we have our database we can start reducing things. We should probably start with the Biases. If the biases we
+want to reduce were observed on 2015-12-22 on spectrograph 2, arm red (r) at site S:
+
+reduceBias.py PFS --calib PFS/CALIB --output PFS --calibId calibVersion=bias arm=r calibDate=2015-12-22 spectrograph=2 --do-exec --id field=BIAS dateObs=2015-12-22 --clobber-config --nodes=1 --procs=1
```

Comment:
This produces lots of output that I didn't expect (list of set up packages, a dump of the environment, etc etc). I guess that's because it's a `BatchPoolTask`? If so, I guess that's an LSST quirk.